### PR TITLE
Flip-Flop Interaction is improved. For issue #95

### PR DESCRIPTION
**Pull request by mohit-003**
### For GCI [Task](https://codein.withgoogle.com/dashboard/task-instances/5679155207733248/) and [issue #95](https://github.com/CircuitVerse/Interactive-Book/issues/95)

I have made the following changes:

- Press **R** for reload 
- Press **Enter** for next
- Press **L** for going back

_I have made the changes, So it would work with both small and capital letters._
**@sal2701**

az
az1
az2
akaaa

**_-By @mohit-003_**

### CODE REVIEW

**Comment by mohit-003** [comment]
Yep Done!

**Comment by mohit-003** [comment]
What should I change the name to? @sal2701

**Comment by mohit-003** [comment]
Done! @sal2701

**Comment by mohit-003** [comment]
I think the mouse should be clicked outside and then n should be pressed.

**Comment by sal2701** [comment]
Okay, but can you fix it? Because it might get irritating at times :grimacing:

**Comment by mohit-003** [comment]
Yes for sure😁

**Comment by mohit-003** [comment]
@sal2701 please help me with that I'm not able to understand on how to do it.

**Comment by mohit-003** [comment]
Yes done👍🏻

**Comment by mohit-003** [comment]
Can you please give me some hint on how to do it?

**Comment by sal2701** [comment]
@mohit-003 The basic idea is to check if the user has pressed n or N and remove the last n from the string. Which is what you've tried to do but your code is not going in to the right loop while slicing the last letter. Check it again.

**Comment by mohit-003** [comment]
Can I use enter or space bar or an arrow key for going next?

**Comment by sal2701** [comment]
Yeah why not, but make sure it doesn't get appended to the string

**Comment by mohit-003** [comment]
Yep finally done!
I did press Enter for next.

File: _includes/flipflop2.html [hunk 1]
```diff
@@ -270,22 +306,35 @@ <h4>Press <strong>f</strong> to begin.</h4>
 						summary += '<p class="form-inline">';
 
 						var keys = Object.keys(flipflops);
-
 						for (var i = 0; i < keys.length; i++) {
+							if (keys[i] !== "Z"){
 							summary += '<h3>' + keys[i] + ' = ' + flipflops[keys[i]] + '</h3> ';
+							}
+							else {
+								if (flipflops[keys[i]][length-1] === "n" || flipflops[keys[i]][length-1] === "N"){
```
Comment (sal2701) [inline _includes/flipflop2.html]: Hey @mohit-003  I don't think we still need these lines

**Comment by mohit-003** [comment]
👍🏻👍🏻

**Comment by aayush-05** [comment]
@mohit-003 can you please fix the merge conflicts ?

**Comment by mohit-003** [comment]
@aayush-05 Can I make a new PR, as I have deleted this fork?

**Comment by aayush-05** [comment]
Sure go ahead @mohit-003

**Comment by mohit-003** [comment]
@aayush-05 I have opened a new [PR](https://github.com/CircuitVerse/Interactive-Book/pull/274)

**Pull request closed without merging**